packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -377,7 +378,7 @@ const Dashboard: FC = () => {
 
     const handleExportDashboard = () => {
         if (!dashboard) return;
-        exportDashboard(dashboard);
+        exportDashboard({ dashboard, queryFilters: location.search });
```

Comment:
Can't we use the `allFilters` that get returned from `useDashboardContext()` here? They're of type `DashboardFilters`

Also, we remove some properties in dashboard filters' URL search params to reduce the URL size. But we then re-format these with `convertDashboardFiltersParamToDashboardFilters` (see `DashboardProvider` l:253-281

So I think that we can just use `allFilters` here instead of `location.search` 🤔

Reply:
I agree the location feels a bit hacky. I want the /minimal page to behave exactly as this dashboard at this very moment. What I really want to copy is the result of this effect , https://github.com/lightdash/lightdash/blob/main/packages/frontend/src/providers/DashboardProvider.tsx#L315.

I think using the location will be more stable than replicating any logic using allFilters, what do you think ?

Reply:
ahh, I see, ok! 
Let's do this then, sounds good 👍 
That means that the `/minimal` page then would just do its thing with the search params and convert accordingly, I believe.